content/docs/command-reference/exp/run.md

```diff
@@ -96,9 +91,13 @@ committing them to the Git repo. Unnecessary ones can be [cleared] with
   parallel. Only has an effect along with `--run-all`. Defaults to 1 (the queue
   is processed serially).
 
-  > Note that since queued experiments are run isolated from each other, common
-  > stages may sometimes be executed several times depending on the state of the
-  > [run-cache] at that time.
+  <admon type="warn">
+
+  `dvc exp run --run-all [--jobs]` is now a shortcut for
+  `dvc queue start [--jobs]` followed by `dvc queue logs -f`. The `--run-all`
+  and `--jobs` options will be deprecated in a future DVC release.
+
+  </admon>
```

Comment:
Maybe this should be under `--run-all` ? Or split into 2 admonitions, even if they're similar.

Reply:
@pmrowla What do you think about moving or splitting this one?

Reply:
I don't really think it's necessary to repeat this block after 2 consecutive options (`run-all` and `jobs` will appear one after another and both options + the info block fit into a single screen on most devices

Reply:
True. But this is much more about `--run-all` than about `--jobs` so I moved it up in https://github.com/iterative/dvc.org/pull/3894/commits/26d28e8b4dae92ff3100b7a8ebf5d2bfbd36272a.